monero-outreach-docs/translations/pt-br/storyboard/blockchain_gender_pt-br.md

```diff
@@ -0,0 +1,78 @@
+
+Eu concordo absolutamente com elas. Precisamos nos concentrar e escrever sobre as conquistas das mulheres, em vez de escolher as diferenças entre os gêneros. Neste espaço, sinto que você é reconhecido pelo que você contribui. É sobre avançar com uma agenda em mente e isso é descentralização, não importa em qual projeto você se inclina mais.
+
+Eu acredito em evitar os rótulos e discussões de gênero e, ao invés disso, focar em encontrar mulheres que estão contribuindo neste espaço e certificando-se de que seu trabalho está exposto. Há muitas mulheres por aí interessadas em criptografia.
+
+** Você acha que existem maneiras em que o projeto ou comunidade Monero é particularmente propício para atenuar a divisão de gênero na indústria de blockchain? **
+
+Eles não discriminam - se você tem uma ideia e parece viável, você receberá apoio. Mais uma vez, eu só posso falar por mim mesma, mas na minha experiência, senti que minhas idéias e sugestões foram levadas a sério pelas pessoas com quem lidei. Os homens no espaço Monero / cripto não foram nada além de acolhedores e solidários.
+
+É claro que você será exposto a trolls e sexismo (isso acontece em todos os ambientes), mas isso tem a ver com o caráter inseguro do indivíduo e não deve ser uma generalização de todos os homens. Na maioria das vezes, se a idéia é boa e melhora a tecnologia ou ajuda a comunidade de alguma forma, ela é apoiada - não importa quem tenha a ideia.
+
+** Qual seria o seu conselho para as mulheres que querem se envolver na indústria de criptografia e, em particular, no projeto Monero? **
+
+Apenas faça! A informação lá fora pode ser extremamente esmagadora, mas se você tomar o tempo, você acabará por entendê-lo. Nesse espaço, você sente que há uma quantidade infinita de informações que você nunca conseguirá alcançar, mas é assim que todos se sentem, e nunca se sentem desencorajados!
+
+Há muito o que fazer e há tanta coisa que você pode fazer. Nunca subestime o seu valor e o que você pode contribuir. Há tantos vídeos, artigos, blogs, posts no reddit e pessoas que você pode conversar para ajudá-lo a entender melhor. Então vá em frente!
+
+_ Você é uma mulher que gostaria de ser entrevistada sobre suas experiências no espaço blockchain e criptomoedas? Adoraríamos ouvir você! Por favor, envie um e-mail para [monicammizzi@gmail.com] (mailto: monicammizzi@gmail.com) para que sua voz seja ouvida._
```

Comment:
Suggestion: "Adoraríamos ouvir sua história! "
" para que você seja ouvida." instead of "sua voz seja ouvida", sounds more natural